plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -122,6 +126,7 @@ class NonMetadataPackage(UnitMixin, FileContentUnit):
     release = mongoengine.StringField(required=True)
     checksum = mongoengine.StringField(required=True)
```

Comment:
They are an important part of the unit key still, and as much as I'd like to, I don't think we're prepared to remove them from the unit key yet. If and when we do, we'd need to figure out some other way to establish uniqueness beyond just NEVRA.

I know it means we have a little duplicate data on the model now, but I don't think we can remove the ``checksum`` and ``checksumtype`` attributes yet.